lib/manage.js

```diff
@@ -165,22 +166,80 @@ export async function manage({meeting, meetings, xmppOptions}) {
       if(stanza.attrs.type === 'groupchat') {
         if(stanza.attrs.from in participants) {
           const nick = participants[stanza.attrs.from].name.replace(/ /g, '_');
-          const transcriptNode = stanza.getChild('json-message');
+          const jsonMessageNode = stanza.getChild('json-message');
           const bodyNode = stanza.getChild('body');
+          let msg;
+          if(jsonMessageNode) {
+            try {
+              msg = JSON.parse(jsonMessageNode.text());
+            } catch(e) {
+              console.error('Bad json-message from ' + stanza.attrs.from);
+              return;
+            }
+          }
 
           // the message is a transcription message
-          if(transcriptNode) {
-            const transcript = JSON.parse(transcriptNode.text());
-            if(transcript.event == 'SPEECH' && !transcript.is_interim) {
-              const commenter = transcript.participant.name.replace(/ /g, '_');
-              const rawMessage = transcript.transcript[0].text + '.';
-              const message = commenter + ': ' +
-                rawMessage[0].toUpperCase() + rawMessage.slice(1);
+          if(msg && msg.event == 'SPEECH' && !msg.is_interim) {
+            const commenter = msg.participant.name.replace(/ /g, '_');
+            const rawMessage = msg.transcript[0].text + '.';
+            const message = commenter + ': ' +
+              rawMessage[0].toUpperCase() + rawMessage.slice(1);
+            if(ircOptions.log) {
+              log({logFileBasePath, nick, message});
+            }
+            ircSay(`${nick}: ${message}`);
+            await xmppSay({info: `${nick}: ${message}`, meeting, xmppClient});
+          }
+
+          // the message is a new poll message
+          if(msg && msg.type === 'new-poll') {
+            if(!Array.isArray(msg.answers)) {
+              console.error('Poll with non-array answers');
+              return;
+            }
+            const pollNum = ++polls.count;
+            polls.byId[msg.pollId] = {num: pollNum, msg, votes: {}};
+            polls.idsByNum[pollNum] = msg.pollId;
+            const numOptions = msg.answers.length;
+            // If the poll is created from IRC, the participant/nick is the
+            // bot, so use senderName instead.
+            const bridged = stanza.attrs.from.endsWith('/cgbot');
+            const name = bridged ? msg.senderName : nick;
+            const message = `${name} created poll #${pollNum} with ${numOptions} options.`;
+            ircSay(message);
+            if(ircOptions.log) {
+              log({logFileBasePath, nick: ircOptions.nick, message});
+            }
+            const message2 = `... Question: ${msg.question}`;
```

Comment:
```suggestion
            const message2 = `POLL: ${msg.question}`;
```

Let's do this to keep with the current syntax for "PROPOSAL/USE CASE/RESOLUTION/etc."

Reply:
Applied: f119cc316b687f204b8265c0d558c4a6c92d9302